app/controllers/samples_controller.rb

```diff
@@ -138,11 +138,25 @@ def index_v2
                  current_power.samples
                end
 
+    # Add total_reads
+    # TODO: (gdingle): this appears to be making n+1 queries
```

Comment:
@tfrcarvalho told me to leave the data queries alone until his refactor is in.

Reply:
same comment as above.